From 01ba72b0b7f9b4c5ef7271ef05094f88ad0c96a8 Mon Sep 17 00:00:00 2001
From: "Anna (navi) Figueiredo Gomes" <navi@vlhl.dev>
Date: Fri, 28 Feb 2025 22:09:36 +0100
Subject: [PATCH] librc: fix loading rc.conf

Passing the wrong variable over, we were trying to load /etc, not
/etc/rc.conf
---
 src/librc/librc-misc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/librc/librc-misc.c b/src/librc/librc-misc.c
index b1b9da46..b309396c 100644
--- a/src/librc/librc-misc.c
+++ b/src/librc/librc-misc.c
@@ -418,7 +418,7 @@ rc_conf_value(const char *setting)
 	}
 
 	xasprintf(&conf, "%s/%s", sysconfdir, "rc.conf");
-	rc_conf_append(sysconfdir);
+	rc_conf_append(conf);
 	free(conf);
 
 	/* Support old configs. */
-- 
2.48.1

